client/src/components/History/HistoryDetails.vue

```diff
@@ -1,129 +1,95 @@
                 />
-
                 <PriorityMenuItem
                     key="share"
                     title="Share or Publish"
                     icon="fas fa-handshake"
                     @click="backboneRoute('/histories/sharing', { id: history.id })"
                 />
-
                 <PriorityMenuItem
                     key="delete-this-history"
                     title="Delete this History"
                     icon="fas fa-trash"
-                    @click="$bvModal.show('delete-history-modal')"
+                    @click="showDeleteModal = !showDeleteModal"
                 />
-
                 <PriorityMenuItem
                     key="purge-this-history"
                     title="Purge this History"
                     icon="fas fa-burn"
-                    @click="$bvModal.show('purge-history-modal')"
+                    @click="showPurgeModal = !showPurgeModal"
                 />
-
                 <PriorityMenuItem
                     key="structure"
                     title="Show Structure"
                     icon="fas fa-code-branch"
                     @click="backboneRoute('/histories/show_structure')"
                 />
-
                 <PriorityMenuItem
                     key="extract-workflow"
                     title="Extract Workflow"
                     icon="fas fa-file-export"
                     @click="iframeRedirect('/workflow/build_from_current_history')"
                 />
-
                 <PriorityMenuItem
                     key="set-permissions"
                     title="Set Permissions"
                     icon="fas fa-user-lock"
                     @click="backboneRoute('/histories/permissions', { id: history.id })"
                 />
-
                 <PriorityMenuItem
                     key="make-private"
                     title="Make Private"
                     icon="fas fa-lock"
-                    @click="$bvModal.show('make-private-modal')"
+                    @click="showPrivacyModal = !showPrivacyModal"
                 />
-
                 <PriorityMenuItem
                     key="resume-paused-jobs"
                     title="Resume Paused Jobs"
                     icon="fas fa-play"
                     @click="iframeRedirect('/history/resume_paused_jobs?current=True')"
                 />
-
                 <PriorityMenuItem
                     key="export-citations"
                     title="Export Tool Citations"
                     icon="fas fa-stream"
                     @click="backboneRoute('/histories/citations', { id: history.id })"
                 />
-
                 <PriorityMenuItem
                     key="export-history-to-file"
                     title="Export History to File"
                     icon="fas fa-file-archive"
-                    @click="backboneRoute(`/histories/${history.id}/export`)"
+                    @click="iframeRedirect('/history/export_archive?preview=True')"
```

Comment:
```suggestion
                    @click="@click="backboneRoute(`/histories/${history.id}/export`)"
```
Maybe a bad merge here ? The `preview` keyword doesn't exist anymore